meilisearch/models/task.py

```diff
@@ -1,6 +1,10 @@
-from typing import Any, List, Dict, Optional
+from __future__ import annotations
+
+from typing import Any, Dict, Optional
```

Comment:
Why we didn't remove `Dict` too in this file?

Reply:
Pydantic, which is what `CamelBase` is using behind the scenes, doesn't like `__future__` style annotations. Doing it this way makes Pydantic happy while still providing the benefits it brings. To the end user it is all the same, they don't see any difference with the different annotation types.